azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/ChatCompositeLocalOptions.java

```diff
@@ -31,6 +31,7 @@
  * @see ChatComposite
  */
 public final class ChatCompositeLocalOptions {
+    private boolean isLaunchingWithUI;
```

Comment:
We need to have a default value set, so if Contoso does not provide any value UI will be launched.